src/components/checkboxes/template.njk

```diff
@@ -18,6 +18,11 @@
   {% endif %}
 {% endfor %}
 
+{% set userDefinedClasses = ' ' + params.classes if params.classes %}
```

Comment:
Checkboxes don't currently support the inline modifier, only radios do.

Reply:
thanks, removed